src/components/themed/Alert.js

```diff
@@ -52,17 +52,17 @@ export const useAlertTypeIcon = (type: AlerType, color: string, size: number, st
 
 export function Alert({ type, title, message, numberOfLines = 2, marginRem, paddingRem, onPress, children }: Props) {
   const theme = useTheme()
+  const margin = sidesToMargin(mapSides(fixSides(marginRem, 0), theme.rem))
```

Comment:
Shouldn't this be set to 1 rem like on the left?

Reply:
Oh, darn! I didn't realize the `useSpacing` hook had 1 and 1 as the defaults.